Playbooks/Reopen-Incdient-With-Incomplete-Tasks/incident-trigger/azuredeploy.json

```diff
@@ -0,0 +1,258 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "metadata": {
+        "title": "Reopen-Incident-With-Incomplete-Tasks",
+        "description": "This playbook will reopen a closed Sentinel incident if there are any incident tasks attached which have not been completed.  In addition, a tag and comment will be added to the incident to call attention to the incomplete tasks.",
+        "version": "1.0.0",
+        "mainSteps": [
+            "The incident tasks are enumerated to determine the completion status, and if incomplete will cause the reopening of the incident."
+        ],
+        "prerequisites": [
```

Comment:
Please add post deployment steps in metadata.

Reply:
Added a post deployment section to metadata